home-assistant-linuxserver.json

```diff
@@ -0,0 +1,50 @@
+{
+	"Home Assistant Linuxserver.io": {
+		"description": "Open source home automation that puts local control and privacy first. <p>Based on the custom docker image (so non-root user can be used): <a href='https://hub.docker.com/r/homeassistant/home-assistant' target='_blank'>https://hub.docker.com/r/linuxserver/homeassistant</a>. Available for amd64, armv7 & arm64.</p><p>Note that this container uses the net=host option to allow for multi-cast and uPnP functionality that Home Assistant provides.</> ",
+		"ui": {
+			"slug": ""
+		},
+		"website": "https://home-assistant.io/",
+		"version": "1.2",
+		"containers": {
+			"homeassistantlsio": {
+				"image": "lscr.io/linuxserver/homeassistant",
+				"tag": "latest",
+				"launch_order": 1,
+				"opts": [
+					[
+						"--net",
+						"host"
+					]
+				],
+				"ports": {
+					"8123": {
+						"description": "Home Assistant UI port. You may need to open the same port on your firewall (protocol: tcp).",
```

Comment:
Maybe we should add a note that it MUST be 8123 (as it's using host networking).

Reply:
Good point, since we are enforcing host mode on this, that makes sense.